packages/safe-ds-eda/src/components/profiling/ProfilingInfo.svelte

```diff
@@ -0,0 +1,116 @@
+<script lang="ts">
+    import type { Profiling, ProfilingDetail } from '../../../types/state';
+
+    export let profiling: Profiling;
+    export let imageWidth: number = 200;
+
+    const getProfilingItemColor = function (profilingItem: ProfilingDetail) {
+        if (profilingItem.interpretation === 'good') {
+            return 'var(--primary-color)';
+        } else if (profilingItem.interpretation === 'error') {
+            return 'var(--error-color)';
+        } else if (profilingItem.interpretation === 'warn') {
+            return 'var(--warn-color)';
+        } else if (profilingItem.interpretation === 'bold') {
+            return 'var(--font-dark)';
+        } else if (profilingItem.interpretation === 'default') {
+            return 'var(--font-light)';
+        } else if (profilingItem.interpretation === 'category') {
+            return 'var(--font-light)';
+        } else {
+            return 'var(--font-light)';
+        }
+    };
+</script>
+
+<div class="wrapper">
+    <div class="profilingItemsTop">
+        {#each profiling.top as profilingTopItem}
```

Comment:
What is the _semantic_ difference between `profiling.top` and `profiling.bottom`? Better use that as the name of those properties in the model. How this is then displayed is dictated by the Svelte components.

Reply:
there is no real semantic difference, it is just what is displayed at top and what at the bottom, separated by the space in between, who's size is dictated by the biggest profiling info.

Reply:
It looks like `top` is always `[validRatio, missingRatio]`, which can be displayed regardless of the type of the column. It seems like the restriction "can be displayed regardless of type" could be used to differentiate what goes to the top and what goes to the bottom.

Thus, I'd suggest replacing e.g.

```ts
profiling: {
    top: [validRatio, missingRatio],
    bottom: [
        { type: 'text', value: 'Categorical', interpretation: 'bold' },
        ...uniqueProfilings,
    ],
},
```

with

```ts
profiling: {
    validRatio,
    missingRatio,
    other: [
        { type: 'text', value: 'Categorical', interpretation: 'bold' },
        ...uniqueProfilings,
    ],
},
```

This way, the model does not already dictate the representation.